cadence/values.yaml

```diff
@@ -370,9 +366,16 @@ web:
 
   affinity: {}
 
-dynamicConfig:
-  values: {}
-  pollInterval: "10s"
+dynamicconfig:
```

Comment:
any reason why the naming convention changed here? there are parts that still refer to the old snakeCase version. can we revert this to `dynamicConfig` before merging